Installers/default.build

```diff
@@ -54,72 +54,9 @@
          </fileset>
       </delete>
       <mkdir dir="root/tmp" />
-      <!-- Build Addin .pkg -->
       <copy file="../IDE/MonoDevelop/MonoDevelop.MonoGame_${buildNumber}.mpack" tofile="./root/tmp/MonoDevelop.MonoGame_${buildNumber}.mpack" overwrite="true"/>
       <exec program="pkgbuild" workingdir="../Tools/Pipeline/bin/MacOS/AnyCPU/Release" commandline=" --component ./Pipeline.app --identifier com.monogame.pipeline --version ${buildNumber} --scripts ${project::get-base-directory()}/MacOS/Scripts/Pipeline --install-location /Applications ${project::get-base-directory()}/Pipeline.MacOS.pkg" />
       <exec program="pkgbuild" workingdir="." commandline=" --root ./root --identifier com.monogame.xsaddin --version ${buildNumber} --scripts ${project::get-base-directory()}/MacOS/Scripts/Addin ${project::get-base-directory()}/MonoGame.XamarinStudio.Addin.pkg" />
-      <!-- Build Framework .pkg -->
-      <delete dir="./root/tmp" />
-      <mkdir dir="root/Library/Frameworks/MonoGame.framework/v${buildNumber}/Tools" />
-      <mkdir dir="root/Library/Frameworks/MonoGame.framework/v${buildNumber}/Assemblies/iOS" />
```

Comment:
Ups, forgot to copy paste changes into new version of the file, I used the file from https://github.com/mono/MonoGame/pull/4177 which had a bit of outdated branche, will fix it right away.